nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Discover/DiscoveryWorkerTests.cs

```diff
@@ -1094,6 +1094,61 @@ await TestDiscoveryAsync(
         );
     }
 
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public async Task DiscoveryReportsDependencyFileNotParseable(bool useDirectDiscovery)
+    {
+        // override temp directory
+        using var tempDir = new TemporaryDirectory();
```

Comment:
`TestDiscoveryAsync()` should create its own temp directory so this might not be necessary.

Reply:
Oops yea I forgot to delete that